modules/mono/editor/bindings_generator.cpp

```diff
@@ -95,7 +95,7 @@
 #define C_METHOD_MONOARRAY_TO(m_type) C_NS_MONOMARSHAL "::mono_array_to_" #m_type
 #define C_METHOD_MONOARRAY_FROM(m_type) C_NS_MONOMARSHAL "::" #m_type "_to_mono_array"
 
-#define BINDINGS_GENERATOR_VERSION UINT32_C(11)
+#define BINDINGS_GENERATOR_VERSION UINT32_C(13)
```

Comment:
Is the bump from 11 to 13 expected?

Reply:
This was back-ported from master so probably version 12 was some other change that wasn't back-ported. It doesn't matter too much as this is used together with the API hash.